flecsi-sp/minimal/mesh.h

```diff
@@ -51,6 +58,10 @@ class minimal_mesh_t
   static constexpr size_t dimension = minimal_config_t::num_dimensions;
   
   using vertex_t = minimal_types_t::vertex_t;
+  using edge_t = minimal_types_t::edge_t;
+#if FLECSI_MESH_DIMENSION == 3
```

Comment:
Where can I define `FLECSI_MESH_DIMENSION` to be `3`? (It is hard-coded to `2` in `specializations/minimal/config.h`.